test_programs/plonky2_trace/2_function_calls/Nargo.toml

```
@@ -0,0 +1,7 @@
+[package]
+name = "2_function_calls"
+type = "bin"
+authors = [""]
+compiler_version = ">=0.31.0"
+
+[dependencies]
```

```suggestion
[dependencies]

```
Shouldn't here have a new line? What is the convention for `.toml` files? Maybe set up a formatter?